src/App.js

```diff
@@ -18,17 +18,29 @@ function App() {
           <Col>
             <Routes>
               <Route path="/" element={<LogIn />} />
-              <ProtectedRoute path="/students"
-                component={Students}
-                isAuth={false}
+              <Route
+                path='/students'
+                element={
+                  <ProtectedRoute user={false}>
```

Comment:
I assume `false` is a placeholder? What will you replace it with?
If it is going to be a `boolean` then `user` is not a great name for the variable. It (to me) implies an object containing user documents.

If it's just a boolean, than `isLoggedIn` would be a more descriptive variable name